src/worker.js

```diff
@@ -367,8 +367,37 @@ const aioli = {
 				}
 			},
 			// Setup print functions to store stdout/stderr output
-			print: text => tool.stdout += `${text}\n`,
-			printErr: aioli.config.printInterleaved ? text => tool.stdout += `${text}\n` : text => tool.stderr += `${text}\n`
+			print: text => {
+				tool.stdout += `${text}\n`;
+				if(this.config.printStream)
```

Comment:
I really don't remember why I've been using `aioli.config` instead of `this.config` but let's keep it the same in case it's important 😀 

```suggestion
				if(aioli.config.printStream)
```